app/jobs/wakes/update_facebook_metrics_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class Wakes::UpdateFacebookMetricsJob < ActiveJob::Base
+  queue_as :wakes_metrics
```

Comment:
@gaganawhad also, why a separate queue?

Right now, GA is throttled to accommodate the rate limit. It is processing as fast as GA will allow, but it results in some worker downtime. We queue up GA workers every 90 seconds, but it should only take about 30 seconds for them to run. Thus, that queue is currently idle about 2/3 of the time.

That said, maybe the way forward is to give them separate queues in `Wakes` to give us flexibility, and then in Heroku we have the same set of dynos work both queues (for now, til we hit problems). What do you think?